From 1a2e71258fe03fc009ca67e5150c6006d834736d Mon Sep 17 00:00:00 2001
Message-ID: <1a2e71258fe03fc009ca67e5150c6006d834736d.1702907703.git.stefan@agner.ch>
From: Andrei Litvin <andy314@gmail.com>
Date: Wed, 13 Dec 2023 13:27:16 -0500
Subject: [PATCH] Bump prompt-toolkit requirement to fix ptpython version
 update (#30987)

---
 scripts/setup/constraints.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/setup/constraints.txt b/scripts/setup/constraints.txt
index 5322b58f7f..892e91964a 100644
--- a/scripts/setup/constraints.txt
+++ b/scripts/setup/constraints.txt
@@ -237,7 +237,7 @@ prettytable==2.5.0
     # via
     #   mbed-ls
     #   mbed-os-tools
-prompt-toolkit==3.0.38
+prompt-toolkit==3.0.43
     # via ipython
 protobuf==3.20.1
     # via
-- 
2.43.0

